src/services/notifications/notifications.hooks.ts

```diff
@@ -1,9 +1,41 @@
 import * as authentication from '@feathersjs/authentication';
 import { setField } from 'feathers-authentication-hooks';
+import validate from 'feathers-validate-joi';
+import {
+  createNotificationSchema,
+  getNotificationFilters,
+  notificationId,
+  updateNotificationSchema,
+} from './notifications.validations';
+import isValidId from '../../hooks/isValidId';
+import { iff, isProvider } from 'feathers-hooks-common';
+import isValidQueryParam from '../../hooks/isValidQueryParam';
+import isExistingUser from '../../hooks/isExistingUser';
+import { HookContext } from '@feathersjs/feathers';
 // Don't remove this comment. It's needed to format import lines nicely.
 
 const { authenticate } = authentication.hooks;
 
+/**
+ * Add isRead = false query param to update only not read notifications
+ * */
+function setNotificationStatus(context: HookContext): HookContext {
+  if (!context.id){
+    context.params.query = {
+      isRead: false,
+      ...context.params.query,
+    };
+  }
+  return context;
+}
+function sortByCreatedAt(context: HookContext): HookContext {
+  context.params.query = {
```

Comment:
Since it's supposed to be default sorting, and we allow sending $sort, we need to check if we already have this param before, otherwise set default one.

```
if(!context.params.query.$sort) { 
 // add default sorting
}
```

Reply:
added 699dbe923c9802070fe9ee9d5f0fe9809ba4a2e2